bin/install_desimodel_data

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+import sys
+from desimodel.install import main
+
+if __name__ == '__main__':
```

Comment:
This is just a matter of style, but you don't need the `if` when it's just a stub executable like this. You can just do a bare `sys.exit(main())`.

Reply:
That's true in this particular case, but in general if the code under `main()` involves any multiprocessing via spawn (default on MacOS), then it needs to be wrapped under an `if __name__ == '__main__':` block so I think it is good practice to always do that.  See https://superfastpython.com/multiprocessing-spawn-runtimeerror/ for more explanation.

Reply:
Ah, did not know that. That's fine then.